test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -184,16 +184,98 @@ GetCondition(const std::string &counter_name, const std::string &then_block)
 	return retval;
 }
 
+/**
+ * Class that registers and provides context during code
+ * generation.
+ * Used to generate correct Lua code.
+ */
+class Context {
+public:
+	enum BlockType {
+		Returnable,
+		Breakable,
+	};
+
+	void step_in(BlockType type)
```

Comment:
Don't mix snake-case and camel-case names styles. Here and below.

Reply:
Here are some conventions regarding naming styles in C++ to improve readability:
1. Variables should be in snake_case;
2. Type names should be in PascalCase;
3. Consts (even ones in enum) should be like `kConstValue`.

It's quite different how function should be named: STL uses snake_case, [Qt uses](https://wiki.qt.io/Qt_Coding_Style) camelCase, [Google](https://google.github.io/styleguide/cppguide.html#Function_Names) and [Yandex](https://github.com/yandex/CMICOT/blob/master/CPP_STYLE_GUIDE.md#variables) use PascalCase.

In my previous PR we agreed to use PascalCase for non-member functions and snake_case for member functions, so in this PR I use these naming style.

Also, I made a mistake by not naming the enum values as constants, but I've now fixed this error.

Reply:
Okay, thanks!